### DISCUSSION TITLE
Where Can I Download YOLOv9t Version?

### DISCUSSION THREAD

**Discussion by aiyafi** [ultralytics/ultralytics#13007] [General]
I've recently come across some performance metrics for different YOLO models, including the YOLOv9t version, which seems to be very efficient in terms of parameter count and FLOPs. Here are the details from the table I found:
![image](https://github.com/ultralytics/ultralytics/assets/63116789/8895d65c-a44d-4184-8d8d-9c75b23e2b6d)
or you can go to: https://docs.ultralytics.com/models/yolov9/#performance-on-ms-coco-dataset:~:text=Table%201.%20Comparison%20of%20State%2Dof%2Dthe%2DArt%20Real%2DTime%20Object%20Detectors

Since it appears to be a new version, I haven't been able to find a source or repository where I can download this specific model. I'm planning to do some research for my thesis and this model would be very beneficial for my work. Does anyone know where I can download the YOLOv9t model? Any help or pointers would be greatly appreciated!

Thank you in advance!

**Comment by glenn-jocher** [comment]
Hello! As of now, only the configurations for `YOLOv9c` and `YOLOv9e` have been released and are available for download. The other model scales, including `YOLOv9t`, have not yet been published. I recommend keeping an eye on the [official YOLOv9 documentation](https://docs.ultralytics.com/models/yolov9/) for updates on when these models become available. Your interest in using YOLOv9 for your research sounds exciting, and I hope the available models will be beneficial for your thesis in the meantime! 😊
